app/controllers/admin_controller.rb

```diff
@@ -418,22 +418,17 @@ def do_anonymize_person
     render 'anonymize_person'
   end
 
-  def finish_unfinished_persons
-    @finish_persons = FinishPersonsForm.new(
-      competition_ids: params[:competition_ids] || nil,
-    )
+  private def competition_list_from_string(competition_ids_string)
+    (competition_ids_string || '').split(',').uniq.compact
```

Comment:
You can make sure that there's always a valid string being passed to this method via stuff like `params.fetch(:foo, 'my_default')`